frontend/app/calculators/transportation/trip-results.js

```diff
@@ -24,6 +24,8 @@ export default class TransportationTripResultsCalculator extends EmberObject {
   }
 
   // Returns an object whose keys are temporal types
+  // TODO: It looks like many of the constructor parameters are used in this CP but are
+  // not added to the computed signature. Add them to this signature.
```

Comment:
Okay thanks, I don't see a constructor on this class, though. Maybe we can document how this class gets instantiated up top?

Reply:
Maybe constructor isn't the right term. I meant the EmberObject parameters documented from lines 6 - 17 in this file